### PR TITLE
Nm 20

### PR DESCRIPTION
**Pull request by SilviaKorcekova**

### CODE REVIEW

File: services/github.service.js [hunk 1]
```diff
@@ -42,8 +42,18 @@ function getCommitsForRepositoryPerUser(userName, repositoryName, startDate, end
         });
 }
 
+function getCommentsPerRepositoryForUser(repositoryName, startDate) {
```
Comment (jjurcik) [inline services/github.service.js]: We want to have startDate, endDate everywhere, or this is something different?

File: settings.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "github": {
     "url": "https://api.github.com",
     "credentials" : {
-      "login": "jjurcik",
-      "token" : "<here put the token>"
+      "login": "SilviaKorcekova",
+      "token" : "8302c5c76e70842b5c0a8bd73430480f2a8a1e97"
```
Comment (jjurcik) [inline settings.json]: Do not commit your real github token. Github will recognize that and disable it for you. :) Let my placeholder.

File: github-spec.yaml [hunk 3]
```diff
@@ -306,6 +342,17 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/user-active-days'
+    user-comments-count:
+      type: object
+      properties:
```
Comment (tjalc) [inline github-spec.yaml]: I think that commitsCount is missing in spec

File: services/github.service.js (hunk 1 above)
Comment (SilviaKorcekova) [reply services/github.service.js]: This change is part of Adrian Molcan ticket :)

File: services/commits.service.js [hunk 4]
```diff
@@ -41,8 +88,35 @@ function testCommitForUser(commit, member) {
...
+        }});
+    return count;
+}
+
 module.exports = {
     getCommits,
     countCommits,
-    getCommitForTeam
+    getCommitForTeam,
+    createRegexp,
```
Comment (tjalc) [inline services/commits.service.js]: do we need to export countCommitsWithoutJiraTicket and createRegex functions?

Comment (SilviaKorcekova) [reply services/commits.service.js]: I rewrited it, you are right we dont, thank you :)

**Pull request merged**